doc/getting_started.rst

```diff
@@ -17,29 +17,162 @@
+    Watching everything is the first step in the editing process.  Writer and 
+    filmmaker David Andrew Stoler says there is gold in the most unlikely of 
+    places: "Some of the most beautiful expressions you're going to get from 
+    the actors are after the cut."
+
+#. Edit for a Story
+    Remember that as you edit, you are telling a story.  Editing is so much
+    more than merely cutting footage and adding effects.  It is an opportunity 
+    to take your audience on a journey.  Whether you are editing a complex 
+    narrative film or only putting together a personal video, you tell a more 
+    in-depth story.
+
+#. Keyboard Shortcuts
+    One of the easiest ways to tell the difference between and professional 
+    video editor and a novice is to simply look at how much they use the 
+    keyboard.  Editors that have been in the business for some time know that a 
+    few seconds saved add up over the length of the project.
+
+#. Learn the Lingo
+    Video editing is not just a hobby or a profession; it is an industry.  And 
+    just like any industry, there is a ton of jargon to learn.  Practically 
+    speaking, you do not need to know all the terms on 
+    `this list <glossary.html#definitions>`_  to become a better video editor, 
+    but a fundamental knowledge of the terms may help you communicate better 
+    with other video editors or clients.
+
+#. Assemble, Then Make a Rough Cut
+    Drag and drop all your video footage into a timeline and make sure your 
+    frame size and frame rates are consistent.  Begin a new timeline and 
+    drag-and-drop the best clips into what becomes your assembly cut.  Remember 
+    to save your work frequently, and notate the date and time of each version.
+    
+#. Refine Your Video
+    In this phase, your rough cut begins to resemble a cohesive project.
+    Adjust the sound and color, make sure the dialog is audible, and add music,
+    titles, or graphics in this phase.  Color correction is the process of 
+    setting your footage to a color baseline.  No matter how great your subject 
+    looks on set, you will almost always need to do some basic post-processing 
+    for a consistent video.
+ 
+#. Refine Some More
+    A slow scene can set the mood and add tension or it can bore an audience.  
+    A fast scene can add adrenaline to your audience’s systems or it can give 
+    them headaches.  Some editors cut their projects several different ways 
+    before they find the right pace.  Do not let cutting your project several 
+    times discourage you.
+    
+Exporting
+---------
+People view most of their projects on phones, tablets, or computers, so it is 
+essential to know how to export for the web.  The goal when exporting a video 
+for the web is to create the highest quality possible with the smallest file 
+size.  Four main factors determine the file size of your finished video:
+
+* Codec: 
+   A codec determines the type of file format (MP4, AVI, MOV).  The more 
```

Comment:
This is not correct. A file format (MP4, AVI, MOV) supports 1 or more codecs. Codecs are used to compress and store video and audio data. Not all MP4's use the same Codec for example.

Reply:
Video size is more related to bitrate and encoding settings... vs a specific Codec